_data/banking.yml

```
@@ -668,6 +828,16 @@ websites:
     img: holvi.png
     twitter: Holvi
     facebook: HolviCom
+    regions:
+      - at
+      - be
+      - fi
+      - fr
+      - de
+      - nl
+      - ir
+      - it
+      - gb
 
   - name: HSBC
     url: https://www.hsbc.com/
```

```suggestion
  - name: HSBC UK
    url: https://www.hsbc.co.uk/
```
This entry specifically references the UK login process. Each country HSBC operates in might have its own login process (there are 64 countries where HSBC operates). It appears most European and Asian countries HSBC operates in use the same online banking system, but US, Canada, Argentina, etc. use a different system.

Change requires reordering manually after accepting.